src/Vensim/VensimParse.cpp

```diff
@@ -224,18 +224,13 @@ static std::string compress_whitespace(const std::string& s)
 	return rval;
 }
 
-bool VensimParse::ProcessFile(const std::string &filename)
+bool VensimParse::ProcessFile(const std::string &filename, const char *contents, size_t contentsLen)
 {
    sFilename = filename ;
-    try {
-      mfSource.open(filename);
-    }
-    catch(...) {
-       return false ;
-    }
-    if(mfSource.is_open()) { 
+
+    if(true) {
```

Comment:
I know its weird to have an `if(true)`, but I didn't want to change the indentation of this whole long block to make this as easy to review as possible